src/Media/ImageCollectionFactory.php

```diff
@@ -109,4 +137,35 @@ public function normalize($link)
         $originalUri = Http::createFromString($link)->withScheme('http');
         return $this->uriNormalizer->__invoke($originalUri);
     }
+
+
+    /**
+     * @param CultureFeed_Cdb_Item_Base $cdbItem
+     * @param Language $language
+     * @return CultureFeed_Cdb_Data_Media
+     */
+    protected function getMedia(CultureFeed_Cdb_Item_Base $cdbItem, Language $language)
+    {
+        $details = $cdbItem->getDetails();
+        $details->rewind();
```

Comment:
Goed gezien dat dit nodig is door de manier waarop de code in `cultuurnet/Cdb` geschreven is, maar hoort misschien beter thuis in `getDetailByLanguage()` zelf zodat anderen niet de fout maken van niet preventief te rewinden?